modules/template/src/main/twirl/scaladex/view/project/artifacts.scala.html

```diff
@@ -10,139 +10,74 @@
 @import scaladex.view.Formats
 
 @import java.time.Instant
+
 @(
     env: Env,
     user: Option[UserState],
     project: Project,
-    header: ProjectHeader,
-    artifactName: Artifact.Name,
-    allBinaryVersions: Seq[BinaryVersion],
-    artifactByVersions: SortedMap[(Instant, SemanticVersion), Seq[Artifact]],
-    params: ArtifactsPageParams
+    header: Option[ProjectHeader],
+    groupedArtifacts: Map[SemanticVersion, Map[Artifact.Name, Seq[Artifact]]]
 )
 
 @main(env, title = project.repository.toString, user) {
   <main id="container-project">
-    @headproject(env, user, project, Some(header), "Artifacts")
+    @headproject(env, user, project, header, "Artifacts")
     <div class="container">
       <div class="content-project artifacts box" data-organization="@project.reference.organization"
       data-repository="@project.reference.repository">
         <div class="artifacts-filter">
-          @filters()
           <div class="result-count">
-            <b>@artifactByVersions.size</b>
-            @Formats.wordPlural(artifactByVersions.size, "version") found for
-            <b>@artifactName</b>
+            <b>@groupedArtifacts.size</b> @Formats.wordPlural(groupedArtifacts.size, "version") found
```

Comment:
I think this should be the number of artifact names instead of the number of versions

Reply:
Done